streaming/src/main/java/io/gearpump/streaming/javaapi/Processor.java

```diff
@@ -36,6 +43,41 @@ public Processor(Class<T> taskClass, int parallelism) {
   }
 
   /**
+   * Create a Source Processor
+   * @param dataSink
+   * @param parallelism
+   * @param description
+   * @param taskConf
+   * @param system
+   * @return
+   */
+  public static Processor<DataSinkTask> sink(DataSink dataSink, int parallelism, String description,  UserConfig taskConf, ActorSystem system) {
```

Comment:
maybe passing in `ClientContext` and hiding `ActorSystem` from users is better ?